docs/static/releasenotes.asciidoc

```diff
@@ -1,16 +1,57 @@
+  This mode is controlled individually with each plugin’s ecs_compatibility option, which defaults to the value of the Logstash pipeline.ecs_compatibility setting.
+  In Logstash 8, this compatibility mode will be on-by-default for all pipelines.
+  If you wish to lock in a pipeline’s behavior from Logstash 7.x before upgrading to Logstash 8,
+  you can set `pipeline.ecs_compatibility: disabled` to its definition in `pipelines.yml` (or globally in `logstash.yml`).
+* Starting from Logstash 8.0, the minimum required version of Java to run Logstash is Java 11.
+  By default, Logstash will run with the bundled JDK, which has been verified to work with each specific version of Logstash,
+  and generally provides the best performance and reliability.
+* Support for using `JAVA_HOME` to override the path to the JDK that Logstash runs with has been removed for this release.
+  In the `8.x` release, users should set the value of `LS_JAVA_HOME` to the path of their preferred JDK if they
+  wish to use a version other than the bundled JDK. The value of `JAVA_HOME` will be ignored.
+* The Java Execution Engine has been the default engine since Logstash 7.0, and works with plugins written in either Ruby or Java.
+  Removal of the Ruby Execution Engine will not affect the ability to run existing pipelines. https://github.com/elastic/logstash/pull/12517[#12517]
+* We have added support for UTF-16 and other multi-byte-character when reading log files. https://github.com/elastic/logstash/pull/9702[#9702]
+* Setting `config.field_reference.parser` has been removed.
+  The Field Reference parser interprets references to fields in your pipelines and plugins.
+  Its behavior was configurable in 6.x, and since 7.0 allowed only a single option: `strict`.
+  8.0 no longer recognizes the setting, but maintains the same behavior as the `strict` setting.
+  {ls} rejects ambiguous and illegal inputs as standard behavior. https://github.com/elastic/logstash/pull/12466[#12466]
+
+For a more detailed view of these changes please check <<breaking-8.0>>.
+
+[[features-8.0.0]]
+==== New features and enhancements
+* As processing times speed up, millisecond granularity is not always enough. Inbound data increasingly has sub-millisecond granularity timestamps.
+  The pull request https://github.com/elastic/logstash/pull/12797[#12797] allows the internal mechanisms of
+  Logstash that hold moment-in-time data - such as the Logstash Event, the Persistent Queue, the Dead Letter Queue and JSON encoding/decoding - to have nanosecond granularity.
+* We have added another flag to the Benchmark CLI to allow passing a data file with previously captured data to the custom test case.
+  This feature allows users to run the Benchmark CLI in a custom test case with a custom config and a custom dataset. https://github.com/elastic/logstash/pull/12437[#12437]
+
+==== Plugins
+
+Logstash 8.0.0 includes the same versions of all bundled plugins as Logstash 7.17.0.
```

Comment:
```suggestion
Logstash 8.0.0 includes the same versions of all bundled plugins as Logstash 7.17.0.
If you upgrade to 7.17 before upgrading to 8.0 (as recommended), you won't see any changes to plugin versions. 
```

Reply:
Both  ideas seems good to me, linking the plugin section of the `7.17.0` or copy and pasting here. I go for the copy path, so that the user has all they need immediately accessible, WDYT?